charts/enterprise-logs/templates/_helpers.tpl

```diff
@@ -75,3 +75,10 @@ Create the app name of enterprise-logs clients. Defaults to the same logic as "e
 {{- end -}}
 {{- end -}}
 {{- end -}}
+
+{{/*
+Create the service URL including port for MinIO.
```

Comment:
nit: I think a URL must have a scheme. I think this endpoint is more technically correct.